### ISSUE TITLE
Missing file speech_roberta

### ISSUE THREAD

**Issue by HGfog** [zjunlp/SPEECH#1]
Hello, I encountered the error "No module named 'speech_roberta'" when I ran the code.Could you please provide a solution?

**Comment by 231sm** [comment]
Hi, [HGfog](https://github.com/HGfog), thanks for your question.

In fact, we didn't select Roberta as our backbone LM in our [paper](https://aclanthology.org/2023.acl-long.21/). Because the performance on Roberta is generally worse than BERT and DistilBERT. 
We only tested on Roberta when we selected the suitable backbone LMs, but not when we conducted the final experiments. 

If you would like to try on the backbone LM as Roberta, I have also uploaded the [speech_roberta.py](https://github.com/zjunlp/SPEECH/blob/main/speech_roberta.py) for you reference. 

Hope the response helpful. If you have further questions, feel free to contact us!
